### PR TITLE
Develop ync

### PR DESCRIPTION
**Pull request by YNC77**
1. add quotations
2. Set up a database via PostgresSql and create 'TestDatabase' and a table 'testtable'
3. connect to the database by using datasource
4. add JDBC_URL on application.yml
5. add 'insert' method and 'queryAllPrice' method to TestDatabase
6. modify textEventHandle to detect '$' and insert value to testtable

### CODE REVIEW

**Comment by YNC77** [comment]
resolve conflicts.

File: application/src/main/java/tw/bjn/pg/Application.java [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import javax.sql.DataSource;
+import java.net.URI;
+import java.net.URISyntaxException;
 
 @SpringBootApplication
 public class Application {
+
```
Comment (lrode061518) [inline application/src/main/java/tw/bjn/pg/Application.java:14]: 不應該放在這裡

File: application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java [hunk 2]
```diff
@@ -2,16 +2,69 @@
...
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+@Slf4j
+@Component
 
-@LineEventHandler
```
Comment (lrode061518) [inline application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java:10]: eventhandler必須註記@LineEventHandler才會被加入handler list

File: application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java [hunk 3]
```diff
@@ -2,16 +2,69 @@
...
+            mQuotations = reader.parse("quotations.yml");
+        } catch (IOException e) {
+//            log.error("幹",e);
+        }
+
+        return mQuotations;
+    }
+
+    public String getReplySentence() {
+        Map<String, List<String>> mQuotations = getQuotations();
```
Comment (lrode061518) [inline application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java:64]: be careful about null pointer exceptions

File: application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java [hunk 4]
```diff
@@ -2,16 +2,69 @@
...
+
+    }
+
+    public Map<String, List<String>> getQuotations() {
+        Map<String, List<String>> mQuotations = null;
+        try{
+            YamlReader<Map<String, List<String>>> reader = new YamlReader<>();
+            mQuotations = reader.parse("quotations.yml");
+        } catch (IOException e) {
+//            log.error("幹",e);
```
Comment (lrode061518) [inline application/src/main/java/tw/bjn/pg/event/handlers/message/TextEventHandler.java:57]: 要有禮貌

**Pull request merged**